tests/integration/servicecatalog/service_test.go

```diff
@@ -107,6 +108,7 @@ var _ = Describe("odoServiceE2e", func() {
 			project = helper.CreateRandProject()
```

Comment:
To avoid preference conflict in parallel run, use format of context and project create
```
context = helper.CreateNewContext()
os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "config.yaml"))
project = helper.CreateRandProject()
```

Also in ```JsutAfterEach``` unset the env var 
```
os.Unsetenv("GLOBALODOCONFIG")
```